Assets/01_Scripts/LevelManager.cs

```diff
@@ -4,15 +4,58 @@
 
 public class LevelManager : MonoBehaviour
 {
+    [SerializeField] GameObject GameOverCanvas;
+    [SerializeField] GameObject LevelScoreCanvas;
```

Comment:
The name of these variables (`GameOverCanvas` and `LevelScoreCanvas`) should start with lower case, could you change them?

Reply:
No


K I will